common/web/gesture-recognizer/src/engine/headless/gestures/specs/gestureModel.ts

```diff
@@ -116,7 +116,7 @@ export interface GestureModel<Type, StateToken = any> {
      * Only takes effect if a model instantly resolves or rejects upon being considered for
      * inclusion in the model.
      */
-    resetOnResolve?: boolean,
+    resetOnInstantFulfill?: boolean,
```

Comment:
I decided to go ahead and rename it because we're also resetting on rejection - and that's the case we're using for the longpress model to fix the underlying issue.